project.clj

```diff
@@ -38,7 +38,7 @@
                  [org.cyverse/cyverse-groups-client "0.1.9"]
                  [org.cyverse/common-cfg "2.8.3"]
                  [org.cyverse/common-cli "2.8.2"]
-                 [org.cyverse/common-swagger-api "3.4.8"]
+                 [org.cyverse/common-swagger-api "3.4.9-SNAPSHOT"]
```

Comment:
Don't forget to bump this to `3.4.9` after it's published.

Reply:
Of course, yeah. Just easier to get it all up for review first, as you know.

Reply:
💯 Just a friendly (pesky) reminder 😅